src/templates/viewerAndTextBlock-template.html

```diff
@@ -0,0 +1,74 @@
+<section class="viewer-and-text-block {{#if (eq alignment 'right')}} right {{/if}}"
+    style="background-color: {{#var background}}{{/var}};">
+    <div class="inner-container" style="min-height:{{height}};padding-top:{{paddingTop}};padding-bottom:{{paddingBottom}};">
+        {{#if (eq alignment "left")}}
+        <div class="viewer-block">
+            <babylon-viewer style="width:100vw;height:100vh;max-width:{{viewer.width}};max-height:{{viewer.height}}"
+                clear-color="{{viewer.clearColor}}"
+                environment-lighting="{{viewer.environment}}"
+                source="{{viewer.source}}"
+                hotspots='{{json viewer.hotspots}}'
+                camera-orbit="{{viewer.cameraOrbit}}"
+                animation-speed="{{viewer.animationSpeed}}"
+                {{#if viewer.animationAutoPlay}}animation-auto-play{{/if}}
+                {{#if (ne viewer.cameraAutoOrbit false)}}camera-auto-orbit{{/if}}>
+            {{#each viewer.annotations}}
+                <babylon-viewer-annotation hotspot='{{this.name}}'>
+                    {{{this.html}}}
+                </babylon-viewer-annotation>
+            {{/each}}
+            </babylon-viewer>
+        </div>
+        {{/if}}
+        <div class="text-block"
+            style="text-align:{{textAlign}};width:{{textWidth}};margin-left:{{marginLeft}};margin-right:{{marginRight}};">
+            {{#if titleColor}}
+                <h2 style="color:{{titleColor}}">{{title}}</h2>
+            {{else}}
+                <h2>{{title}}</h2>
+            {{/if}}
+            {{#if subtitle}}
+                {{#if subtitleColor}}
+                    <h3 style="color:{{subtitleColor}}">{{subtitle}}</h3>
+                {{else}}
+                    <h3>{{subtitle}}</h3>
+                {{/if}}
+            {{/if}}
+            <p style="color:{{color}}">
+                <script>
+                    var descText = "{{desc}}"; 
+                    var replaceLT = descText.replace(/&lt;/gi, "<");
+                    var result = replaceLT.replace(/&gt;/gi, ">");
+                    var pTags = document.getElementsByTagName('p');
```

Comment:
any reason not to use const here?

Reply:
No, it just came with a copy/paste from imageAndTextBlock-template.html, but I don't mind updating it to `const`.